homeassistant/components/clickatell/notify.py

```diff
@@ -31,10 +35,16 @@ def __init__(self, config):
         """Initialize the service."""
         self.api_key = config[CONF_API_KEY]
         self.recipient = config[CONF_RECIPIENT]
+        self.sender = config[CONF_SENDER]
 
     def send_message(self, message="", **kwargs):
         """Send a message to a user."""
-        data = {"apiKey": self.api_key, "to": self.recipient, "content": message}
+        data = {
+            "apiKey": self.api_key,
+            "to": self.recipient,
+            "from": self.sender,
+            "content": message
+        }
 
         resp = requests.get(BASE_API_URL, params=data, timeout=5)
```

Comment:
Please extract the device specific interface code and data parsing to a standalone library published on PyPI.

https://developers.home-assistant.io/docs/creating_component_code_review#4-communication-with-devicesservices

Reply:
i'm not a maintainer of this code, i'm just fixing a bug. IMHO it is good to address refactoring to the the owner (@dclobato)

Reply:
> i'm not a maintainer of this code, i'm just fixing a bug. IMHO it is good to address refactoring to the the owner (@dclobato)

I'm not the owner.  I'm just a user